docs/model-monitoring/applications.md

```diff
@@ -112,3 +112,42 @@ It is recommended to specify the exact version of the `evidently` package for re
 ```
 
 See a full example in [Realtime monitoring and drift detection](../tutorials/05-model-monitoring.ipynb#evidently-app).
+
+## Scaling
+
+The model monitoring application can be scaled to handle increased workloads through two key dimensions:
+
+1. **Workers**: Individual processes within each Kubernetes pod
```

Comment:
- **Workers**: Individual processes within each Kubernetes pod
- **Replicas**: The number of Kubernetes pods deployed